packages/hooks/src/useOnKeyDown/useOnKeyDown.mdx

```diff
@@ -0,0 +1,75 @@
+---
+name: useOnKeyDown
+menu: Hooks
+route: /hooks/use-on-key-down
+---
+
+import { Playground } from "docz";
+import { useState } from "react";
+import { useOnKeyDown } from "./useOnKeyDown";
+import { Text } from "@jobber/components/Text";
+import { Card } from "@jobber/components/Card";
+import { Content } from "@jobber/components/Content";
+
+# UseOnKeyDown
+
+`useOnKeyDown` is a simple hook that adds the keydown event handler when the
+component is mounted and removing when unmounted.
+
+`useOnKeyDown` should **only** be used when building keyboard shortcut in a
+component.
+
+```tsx
+import { useOnKeyDown } from "@jobber/hooks";
+```
+
+## With a list of Keys
+
+You can specify a list of key's to watch with the event handler.
+
+<Playground>
+  {() => {
+    const initialListText = "Press shift or enter";
+    const [listText, setListText] = useState(initialListText);
+    useOnKeyDown(
+      e => {
+        setListText("You pressed '" + e.key + "'");
+      },
+      ["Shift", "Enter"]
+    );
+    return (
+      <Card title="Shift or Enter Example">
+        <Content>
+          <pre>{listText}</pre>
```

Comment:
```suggestion
        <Content>
        Press shift or enter.
          <pre>{listText}</pre>
```

Thought we could keep the instructions around here. Or, we could also use the card title?